README.md

```diff
@@ -287,25 +291,25 @@ Asset information
 }
 ```
 - `category` Asset type
-- `decimals` The decimal place of the issued asset, which indicates how many digits are reduced to the decimal point. The default is 18 digits.
-- `totalSupply` The total amount of assets issued, when the assets are issued and destroyed, the total amount of assets will change in real time, the total amount of assets issued is `issuance*10^decimals`
-- `isReplenishable` Whether the asset can be replenish, The asset cannot be modified again after set， set when the asset is created, the default is `true`
+- `decimal` The decimal place of the issued asset, which indicates how many digits are reduced to the decimal point. The default is 18 digits.
+- `totalSupply` The total amount of assets issued, when the assets are issued and destroyed, the total amount of assets will change in real time, the total amount of assets issued is `issuance*10^decimal`
+- `isReplenishable` Whether the asset can be replenish. It is not changeable after setup when creating the asset. The default is `true`
 - `isDivisible` Whether the asset is divisible, set when the asset is created, the default is `true`
-- `issuer` The publisher address, based on the signature field at the time the asset was created
+- `issuer` The publisher address. It is not settable
 - `profile` Additional information about the asset
-    - `name` The name of the asset when it was created
+    - `name` The name of the asset
     - `symbol` Asset identification
     - `description` Basic information of assets
-    - `suggesteGasLimit` Trading consumption limit, and ` gasLimit ` same usage, creating assets set by users themselves, the default is 60000
-    - `stop` Whether to stop the special deals, the default is ` false `, if there is a loophole, set it to ` true ` can stop any special deal
+    - `suggesteGasLimit` Suggested gas limit. It is like the `gasLimit` field in transaction. The default is 60000
+    - `stop` Whether to stop all actions of the asset. The default is ` false`
 
 <a name="data-asset-category"></a>
 
 category | description
 --- |---
-1 |  TokenAsset, ERC20 for ethereum
-2 |  NonFungibleAsset, ERC721 for ethereum
-3 |  CommonAsset, Divisible assets, merging the above two scenarios
+1 |  TokenAsset, similar to ethereum's ERC20 token, it can set whether additional issuance is allowed in the future when issuing, and users can divide and transfer money at will after receiving the coin
```

Comment:
1 |  TokenAsset. Similar to Ethereum's ERC20 token. The issuer can decide whether additional issuance is allowed in the future when issuing. The token is divisible.
2 |  NonFungibleAsset. Similar to Ethereum's ERC721 token, it can store a certain amount of information. Every token is indivisible
3 |  CommonAsset. More flexible digital assets. It is suitable for more complex scenarios. Actualy, the TokenAsset and NonFungibleAsset are two special CommonAsset